contracts/RadspecRegistry.sol

```diff
@@ -156,27 +149,46 @@ contract RadspecRegistry is IArbitrable, AragonApp {
         beneficiary = _beneficiary;
     }
 
+    /**
+    * @dev Set the fee for upserting.
+    * @param _upsertFee The beneficiary of registry fees
+    */
+    function setUpsertFee(uint256 _upsertFee)
+        external
+        auth(SET_UPSERT_FEE_ROLE)
+    {
+        upsertFee = _upsertFee;
+    }
+
     /**
      * @dev Upsert a registry entry with a stake.
-     * @param _scope
-     *        The scope of the entry.
-     *        If the scope is the zero address, then the entry is global.
-     * @param _sig
-     *        The signature of the method the entry is describing.
-     * @param _cid
-     *        The IPFS CID of the file containing the Radspec description.
+     * @param _scope The scope of the entry. If the scope is the zero address, then the entry is global.
+     * @param _sig The signature of the method the entry is describing.
+     * @param _cid The IPFS CID of the file containing the Radspec description.
      */
-    function stakeAndUpsertEntry(address _scope, bytes4 _sig, string _cid) external payable;
+    function stakeAndUpsertEntry(address _scope, bytes4 _sig, string _cid)
+        external
+        payable
+    {
+        require(msg.value == upsertFee, ERROR_INCORRECT_STAKE_AMOUNT);
```

Comment:
`upsertFee` should probably be a `minimumUpsertFee` as the fee for *updating* an existing entry should be larger than the previous stake - i.e. if i create an entry with 1 eth as stake, then the fee for updating that entry is some arbitrary amount larger than 1 eth.

Reply:
Ah I haven't implemented this behaviour. So the fee for inserting a new entry should be `>= minimumUpsertFee` and the fee for updating an entry should be `> currentEntry.stake`?

Reply:
Yes.